dask_match/expr.py

```diff
@@ -1107,6 +1111,32 @@ class Fused(Blockwise):
     def _meta(self):
         return self.exprs[0]._meta
 
+    def _tree_repr_lines(self, indent=0, collapse_fusion=True):
+        # Start with fused expressions
+        lines = []
+        root = self.exprs[0]
+        if collapse_fusion:
+            root_id = f"Fused with {len(self.exprs) - 1} expressions"
```

Comment:
I like the `Fused with` annotation you have now.  I agree that it's nice to distinguish these a little from the others.  I wonder if there is some more visually striking way to do this.  My first attempt at this tried to use `-` rather than ` ` as the indentation character (but only as far left as the fused object was) so it would look like this:

```
Sum
  Fused
  --Add
  ----Mul
  ------ df
```

Maybe that's too much though. Maybe just one dash?  Maybe dots `.`?  Just an idea, not a constraint.

Reply:
Or maybe we include a vertical line

```
Sum
  Fused
  | Add
  |   Mul
  |     df
```

Reply:
I went with the vertical-line idea since it seemed easier to read/test in my experiments. Thanks for the suggestion!